pinot-common/src/main/java/org/apache/pinot/common/restlet/resources/EndReplaceSegmentsRequest.java

```diff
@@ -37,6 +37,10 @@ public class EndReplaceSegmentsRequest {
   private final List<String> _segmentsTo;
   private final Map<String, String> _customMap;
 
+  public EndReplaceSegmentsRequest(@Nullable List<String> segmentsTo) {
+    this(segmentsTo, null);
+  }
+
   public EndReplaceSegmentsRequest(@JsonProperty("segmentsTo") @Nullable List<String> segmentsTo,
```

Comment:
AFAIK we need to add `@JsonCreator` on each class where we have two constructors.